planex/cmd/createmockconfig.py

```diff
@@ -32,6 +32,15 @@ def __call__(self, parser, namespace, values, option_string=None):
         if '=' not in values:
             raise ValueError("invalid option: "+values)
         key, value = values.split('=', 1)
+
+        # convert to data types
+        if value == 'None':
+            value = None
+        if value in ('True', 'False'):
+            value = (value == 'True')
```

Comment:
You can use bool(distutils.utils.strtobool(value)) here.   It will handle upper and lower case and alternatives such as 'yes' and 'no'.

https://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python